app/views/layouts/components/_item.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="govuk-grid-row govuk-!-padding-top-2 govuk-!-padding-bottom-2">
```

Comment:
`item` feels like a non-descriptive name for this component :thinking: not sure what else it could be though

Reply:
We renamed it to `resource_link_with_icon` in https://github.com/DFE-Digital/single-place-for-education-frontend/pull/21/commits/b06800dc96b55c37cbdb7c00053ed78fcc67c9b1. What do you think?